src/OrbitQt/main.cpp

```diff
@@ -266,8 +266,12 @@ int main(int argc, char* argv[]) {
   absl::SetFlagsUsageConfig(absl::FlagsUsageConfig{{}, {}, {}, &orbit_core::GetBuildReport, {}});
   absl::ParseCommandLine(argc, argv);
 
-  InitLogFile(Path::GetLogFilePathAndCreateDir());
+  InitLogFile(Path::GetLogFilePath());
   LOG("You are running Orbit Profiler version %s", orbit_core::GetVersion());
+  ErrorMessageOr<void> remove_old_log_result = TryRemoveOldLogFiles(Path::CreateOrGetLogDir());
+  if (remove_old_log_result.has_error()) {
+    LOG("Fail to remove some old log files:\n%s", remove_old_log_result.error().message());
```

Comment:
nit: maybe "Warning: Unable to remove some old log files: "?

Reply:
Done! Thanks!